app/code/Magento/QuoteGraphQl/Model/Resolver/CartItemPrices.php

```diff
@@ -106,4 +126,35 @@ private function getDiscountValues($cartItem, $currencyCode)
         }
         return null;
     }
+
+    /**
+     * Get Current Currency
+     *
+     * @return string
+     */
+
+    private function getCurrentCurrency()
+    {
+        return $this->storeManager->getStore()->getCurrentCurrencyCode();
+    }
+
+    /**
+     * Get current currency price
+     *
+     * @param string $price
+     * @return string
+     */
+    public function setConvertPrice($price)
```

Comment:
Please declare types for params and return.

Reply:
How to rich you in Slack ?